packages/yew-macro/Cargo.toml

```diff
@@ -21,7 +21,9 @@ lazy_static = "1"
 proc-macro-error = "1"
 proc-macro2 = "1"
 quote = "1"
-syn = { version = "1", features = ["full", "extra-traits"] }
+syn = { version = "1", features = ["full", "extra-traits", "visit-mut"] }
+once_cell = "1"
+prettyplease = "0.1.1"
```

Comment:
Pretty new library, not a lot of downloads, although the author is pretty well known ofc. As far as I see, it's only used in one place, are you sure this dep is needed?

Reply:
You can try remove it and see what happens?